pyproject.toml

```diff
@@ -1,12 +1,8 @@
 [build-system]
 requires = [
-    "numpy >= 1.16.6;python_version<'3.8'",
```

Comment:
This requirement is changed by the adoption of `oldest-supported-numpy`. Unfortunately, I am still stuck with Python 3.7 for the moment.

Perhaps there's an alternative way of doing this that lets us continue to use `oldest-supported-numpy `for the remaining Python versions?

Reply:
I have to admit I can't remember why we have 1.16.6 as the build requirement. Did you go through the previous issues / PRs to confirm this was appropriate?

Reply:
The dependencies for `oldest-supported-numpy` are roughly the same that the old ones (at least for "usual" platforms), except for 3.7. `oldest-supported-numpy` specifies 1.14.5 instead of 1.16.6.

1.16.6 comes from the fact that when I looked for supported packages, I did not look before the install dependency (1.16). As it ignores the install dependency, `oldest-supported-numpy` goes back to the real oldest compatible.

Practically, this does not change anything and I checked that AT builds and runs perfectly on python 3.7. I remind that numpy 1.14.5 is used only for building and is not related to the version used for running AT.

The full explanation is given [here](https://numpy.org/doc/1.24/dev/depending_on_numpy.html). In particular:

 _"…you must build your own binaries (wheels or other package formats) against the lowest NumPy version that you support (or an even older version)_

We are now in the case of _an older version_ !

Reply:
Good, that's seems fine, then.